src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -374,8 +401,18 @@ def get_suggestion(self, random_search=False):
         total_params : dict
             parameter suggestion
         """
-
-        rval = self.rval
+        if self.parallel == True and len(self.running_data):
```

Comment:
Suggest changing "if self.parallel and len(self.runing_data)" to avoid pylint warning.

Reply:
agree